lib/components/user/monitored-trip/trip-monitored-days.tsx

```diff
@@ -0,0 +1,78 @@
+import { FormattedList, FormattedMessage } from 'react-intl'
+
+import FormattedDayOfWeek from '../../util/formatted-day-of-week'
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+import React from 'react'
+
+import getBaseColor from '../../util/base-color'
+
+import styled from 'styled-components'
+
+interface Props {
+  days: string[]
+}
+
+const baseColor = getBaseColor()
+
+const DayCircleContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`
+
+const MonitoredDay = styled.span<{ monitored: boolean }>`
+  align-items: center;
+  background-color: ${(props) => (props.monitored ? baseColor : 'transparent')};
+  border: 1px solid ${(props) => (props.monitored ? baseColor : '#333')};
+  border-radius: 50%;
+  color: ${(props) => (props.monitored ? 'white' : 'inherit')};
+  display: flex;
+  height: 27px;
+  justify-content: center;
+  opacity: ${(props) => (props.monitored ? 1 : 0.7)};
+  text-transform: capitalize;
+  width: 27px;
+`
+
+const daysOfWeek = [
+  'monday',
+  'tuesday',
+  'wednesday',
+  'thursday',
+  'friday',
+  'saturday',
+  'sunday'
+]
+
+const MonitoredDays = ({ days }: Props) => {
+  const monitoredDaysList = (
+    <FormattedList
+      type="conjunction"
+      value={days.map((d) => (
+        <FormattedDayOfWeek day={d} key={d} />
+      ))}
+    />
+  )
+  const DayCircles = daysOfWeek.map((d) => {
+    const dayAbbrev =
```

Comment:
Can you update the i18n abbreviations and use them instead? (They will appear in monitored trip editing too.) Right now, the i18n abbreviations are are 3-letter.